shopify_function_macro/src/lib.rs

```diff
@@ -326,25 +386,45 @@ const OUTPUT_QUERY_FILE_NAME: &str = ".output.graphql";
 /// modules generate Rust types from the GraphQL schema file for the Function input
 /// and output respectively.
 ///
-/// The macro takes two parameters:
+/// The macro takes the following parameters:
 /// - `query_path`: A path to a GraphQL query, whose result will be used
 ///    as the input for the function invocation. The query MUST be named "Input".
-/// - `schema_path`: A path to Shopify's GraphQL schema definition. You
-///   can find it in the `example` folder of the repo, or use the CLI
-///   to download a fresh copy (not implemented yet).
+/// - `schema_path`: A path to Shopify's GraphQL schema definition. Use the CLI
+///   to download a fresh copy.
+/// - `extern_enums`: An list of enums for which an external type should be used.
+///   For those, code generation will be skipped. This is useful to skip
+///   codegen for large enums, or share enums between multiple functions.
```

Comment:
```suggestion
///   codegen for large enums, or share enums between multiple targets.
```

Reply:
Makes sense -- I'll re-use the exact same thing as the README though, for consistency. Not sure why they diverged in the first place.